src/main/metabase/examples.ts

```diff
@@ -199,7 +199,7 @@ function columnExampleQueries(field: Field, table: Table, aidColumns: string[]):
         }
       }
     } else if (numberFieldTypes.includes(field.database_type) && field.fingerprint) {
-      if (field.fingerprint.global['distinct-count'] && field.fingerprint.global['distinct-count'] < 10) {
+      if (field.fingerprint.global['distinct-count'] && field.fingerprint.global['distinct-count'] < 300) {
```

Comment:
I don't agree with this. For text censoring is useless because we don't know if there's a hierarchical pattern, not because of the threshold. For numeric columns 300 bins is way too many. I'd argue that even 10 is too high in some cases.

Reply:
not clear: are you against only the bump for the numeric field case or both?

Reply:
For number I say 10. For text I'm inclined to say never unless we know that there's a pattern in values..

Reply:
Ok with 300 as "almost never" until we figure something else out.

Reply:
Since making examples takes a crap load of time anyway, what do you think about running some queries against postgres during this process to remove some of the guesswork?

Reply:
we're using average length already, so the formula would become somewhat complicated and hard to calibrate. Right now we're just guessing that showing `ceil(1/4 of characters)` might yield an interesting result.

Reply:
How about:

1. If column has few values, `GROUP BY`.
2. If column is hierarchical, introduce substrings.
3. Skip examples for this column altogether.

Reply:
sorry, I've only seen notifications now, but I guess we're OK with the shape of this PR as first baby step, so merging.